package.json

```diff
@@ -21,7 +21,7 @@
     "http-server": "^0.11.1"
   },
   "devDependencies": {
-    "@tableau/extensions-api-types": "1.3.1",
+    "@tableau/extensions-api-types": "1.4.0",
```

Comment:
also, I found one of the reasons why the update command wasn't working. It was missing the "^" in the version. 
Could you change the version to "^1.4.0"?